docs/platform/howto/billing-assign-projects.rst

```diff
@@ -0,0 +1,24 @@
+Assign projects to billing groups
+=================================
+
+When you create a new project, you can assign it to a billing group, which lets you :doc:`use the same payment card and consolidate costs</docs/platform/concepts/billing-groups>`. 
+
+You can also assign the project to another billing group later in the `Aiven Console <https://console.aiven.io/>`_. 
```

Comment:
I'd keep these sentences in one paragraph and append information what the user can actually learn from this article.